nixos/modules/services/matrix/mjolnir.nix

```diff
@@ -81,6 +88,17 @@ in
       default = null;
       description = ''
         File containing the matrix access token for the `mjolnir` user.
+
+        Required when neither encryption or pantalaimon is enabled.
+      '';
+    };
+    passwordFile = lib.mkOption {
+      type = with lib.types; nullOr path;
+      default = null;
+      description = ''
+        File containing the matrix password for the `mjolnir` user.
+
+        Required when encryption or pantalaimon is enabled.
```

Comment:
The move of `passwordFile` to the top-level is nice but we need at least a `lib.mkRenamedOptionModule` to make users aware of the change.

Reply:
oh right, good catch